### PR TITLE
Damian Schenkelman - Story

### PR DESCRIPTION
**Pull request by dschenkelman**
Hey there, I'm submitting my story. Looking forward to getting some feedback.

### CODE REVIEW

**Comment by lethain** [comment]
Hi @dschenkelman -- thanks for submitting. I'll get you feedback over this weekend

File: src/markdown/stories/damian-schenkelman.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: "Damian Schenkelman - Principal Engineer at Auth0"
+name: "Damian Schenkelman"
+role: "Principal Engineer at Auth0"
+slug: "/stories/damian-schenkelman"
+date: "2020-08-05"
```
Comment (lethain) [inline src/markdown/stories/damian-schenkelman.md:6]: Mind updating date to the day you're able to make these changes +2 days? So maybe "2020-08-11" or some such!

File: src/markdown/stories/damian-schenkelman.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+kind: "story"
+---
+<span class="date">August, 2020</span>
+[blog](https://yenkel.dev),
+[twitter](https://twitter.com/dschenkelman),
+[linkedin](https://www.linkedin.com/in/damianschenkelman/)
+
+**_Tell us a little about your current role: where do you work, your title and generally the sort of work do you and your team do._**
+
+I'm a Principal Engineer at [Auth0](https://auth0.com/), an Identity as a Service platform. I work in the Systems Architecture group, which today has three Principal Engineers. We work with different teams on strategic initiatives and also shape Auth0's technical strategy, architecture decisions, and guidelines.
```
Comment (lethain) [inline src/markdown/stories/damian-schenkelman.md:16]: Potentially work linking "technical strategy" to your related post http://yenkel.dev/posts/achieving-alignment-and-efficiency-…

File: src/markdown/stories/damian-schenkelman.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+At the time of writing, I am working with a group of IAM teams as a tech lead of a large new product feature, as well as driving reliability and scaling related initiatives with other teams.
```
Comment (lethain) [inline src/markdown/stories/damian-schenkelman.md:18]: Could you expand the "IAM" acronym? I'm not quite sure what it means

Comment (lethain) [reply src/markdown/stories/damian-schenkelman.md:18]: (From below I guess it's "Identity & Access Management")

File: src/markdown/stories/damian-schenkelman.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+DNA has 6 members (3 permanent Principal, and 3 Staff/Senior II that rotate every 6 months). The workgroup defines decisions and guidelines to help drive Auth0's technology in a specific direction (e.g. avoid language proliferation so we can build libs once and people can switch teams easily) and also collaborate with teams in technical reviews of large initiatives.
+
+I think the biggest way in which my roles differ is that both because I have been at the company for 6+ years and I was also a Director of Engineering, I have the "broadest scope". I work with both Product and Platform teams on initiatives and also work often with other parts of the company: joining conversations with high profile prospects, working with our legal team on contract language, or collaborating with Marketing.
```
Comment (lethain) [inline src/markdown/stories/damian-schenkelman.md:28]: could you clarify if your Director of Eng role was at Auth0 or at prep company?

File: src/markdown/stories/damian-schenkelman.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+On Thursdays (and as much of I can on Tuesdays) I spend my time thinking about:
+- current initiatives and how they are going
+- what we could/should be doing in the future (next quarter, next year)
+- writing docs, guidelines, blog posts
+- (not often) doing POCs and/or writing small tools
+
+**_Where do you feel most impactful as a Staff-plus Engineer? A specific story would be grand._**
+
+The biggest impact comes from being able to help achieve "people scale", positively influencing the work of as many people as possible internally. The book [Scaling Up Excellence](https://www.amazon.com/dp/B00EGMQIDG/ref=dp-kindle-redirect?_encoding=UTF8&btkr=1) provides an easy to understand analogy: scaling is a ground/trenches, war, not a one-off airstrike. It requires a lot of time, and patience but to get to your goals you need to align the whole company in terms of goals and how to get to them.
```
Comment (lethain) [inline src/markdown/stories/damian-schenkelman.md:61]: Extra comma in "ground/trenches, war", might be clearer as "ground war"

**Comment by dschenkelman** [comment]
> @dschenkelman -- overall I think this is fantastic and very excited to add it. I have a few small-ish suggestions that I think would improve readability a bit, if you wouldn't mind taking a look then I'll get it merged!

Thanks, will update now.

**Comment by dschenkelman** [comment]
@lethain just applied the feedback (thanks a lot for the fast reply :)) and squashed.

**Comment by lethain** [comment]
@dschenkelman thanks for the quick turn around! I've merged in, and I'll deploy it out tomorrow after 9AM PT (so that it doesn't get sent to email list until the 10th)

**Pull request merged**